src/Kanbas/Courses/Quizzes/List/index.tsx

```diff
@@ -0,0 +1,12 @@
+import QuizDetails from "../Details";
+
+function QuizList() {
+    return (
+        <>
+            {/* TODO - call QuizDetails to handle Quiz Selection */}
+            <QuizDetails />
```

Comment:
Convert this to a link using Navigate To instead of rendering it right away. This way we don't confuse it for actual functionality and accidentally write styles for it. I also don't want this component to render here if it isnt going to happen in the real scenario.

Reply:
Same reason here, but adding navigate now.